app/controllers/keywords_controller.rb

```diff
@@ -10,6 +10,21 @@ def index
            })
   end
 
+  def show
+    keyword = Keyword.find(params['id'])
+
+    search_entries_query = KeywordSearchEntriesQuery.new.with_keyword(keyword.id)
```

Comment:
When looking at its usage, it feels more natural to do:

```suggestion
    search_entries_query = KeywordSearchEntriesQuery.new(keyword_id: keyword.id)
```

Reply:
👍 . Fixed in 225f90acf4ca3b86aab5e99604d7629d42d8d3da